src/main/java/uk/gov/companieshouse/web/accounts/service/smallfull/impl/LoansServiceImpl.java

```diff
@@ -82,10 +84,15 @@
 
         String uri = LOANS_URI.expand(transactionId, companyAccountsId).toString();
 
+        if(StringUtils.isBlank(addOrRemoveLoans.getLoanToAdd().getDirectorName()) || addOrRemoveLoans.getLoanToAdd().getDirectorName().equals("Prefer not to say")) {
```

Comment:
Could you assign addOrRemoveLoans.getLoanToAdd().getDirectorName() to a temporary variable and use that? It would avoid calling it twice and would make the code more readable.

Reply:
Changed code to assign the above line to a variable and used that variable